src/main/java/com/microsoft/jenkins/kubernetes/wrapper/ResourceManager.java

```diff
@@ -6,23 +6,21 @@
 
 package com.microsoft.jenkins.kubernetes.wrapper;
 
-import com.microsoft.jenkins.kubernetes.util.CommonUtils;
 import com.microsoft.jenkins.kubernetes.util.Constants;
 import io.kubernetes.client.ApiException;
 import io.kubernetes.client.models.V1ObjectMeta;
 import org.apache.commons.lang.StringUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.http.HttpStatus;
 
-import java.io.IOException;
+import java.io.PrintStream;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
 public abstract class ResourceManager {
-    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private PrintStream consoleLogger = System.out;
```

Comment:
change `logger` to  `consoleLogger` 
Print to jenkins console

Reply:
Why do we need this change? `Logger` works fine with Jenkins system.

Reply:
The original logger will print the log to the Jenkins system log, which prints the content to the Jenkins pipeline.